forte/data/entry_type_generator.py

```diff
@@ -39,3 +49,25 @@ def get_type_attributes():
 
```

Comment:
1. Missing return type.
2. I think we need to rewrite the docstring a bit. First, we need to make sure we include  `Return`. And we also need to explain the behavior of this function. For example, it might be important that we mention this actually relies on type annotation.

Reply:
This script is created by @qinzzz. Can I delete something like [get_type_attributes()](https://github.com/asyml/forte/blob/366692ba6edfdbcf4b2738cca75ff976b16c456e/forte/data/entry_type_generator.py#L19) function, cause we already changed  the issue's description.

Reply:
yes, please add typing and modify docsting accordingly. When @qinzzz drafted the class she didn't know the final outcome. So now is the time to finalize them.